.github/workflows/cypress.yml

```diff
@@ -15,6 +15,6 @@ jobs:
         uses: cypress-io/github-action@v2
         with:
           working-directory: ./web
-          start: yarn develop
+          start: yarn develop, yarn run mockserver
           record: true
           wait-on: "http://localhost:8080"
```

Comment:
Do we need another wait-on, or is it fine? (The mockserver should always be available before the dev env is spinning up though) :)

Reply:
That was my thought as well, so I don't think it's necessary, but if it ever fails we can add it?